templates/android/tests/TestException.kt.twig

```diff
@@ -0,0 +1,15 @@
+package tests
+
+import {{ sdk.namespace | caseDot }}.exceptions.ErrorType
+import org.junit.jupiter.api.Assertions.assertEquals
+import org.junit.jupiter.api.Test
+
+class TestException {
+
+    @Test
+    fun testEnumValues() {
+    {% for key, value in testData.enumData %}
```

Comment:
can you generate this off of the spec instead of using `testData`?

Reply:
We are generating this off the spec, it's just that it is not hard coded in all of the tests indivisually.

Reply:
the testData is created from the specs

Reply:
@stnguyen90  I'm a bit confused here, what is meant by generating this of the spec, do we want to generate this from the spec that will be passed to the SDK,
so do we use [$sch['errorTypes'] ](https://github.com/appwrite/sdk-generator/pull/724/files#diff-cceb0b27fc5b234e68eb21429aaf4530b97899ae1489825b484f1391dacc5510R315) to create the tests as well as enum both?

Reply:
> so do we use [$sch['errorTypes'] ](https://github.com/appwrite/sdk-generator/pull/724/files#diff-cceb0b27fc5b234e68eb21429aaf4530b97899ae1489825b484f1391dacc5510R315)to create the tests as well as enum both?

Yes.